<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceDataManagementPage</class>
 <widget class="QWidget" name="MaintainceDataManagementPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
		background-color:black;
}
QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 128);
	border-top: 2px solid white;	
	border-left: 2px solid white;
border-radius:8px;
}</string>
  </property>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>904</x>
     <y>620</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
		border-radius:8px;
background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>610</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>数据记录时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>累计行驶距离</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU1牵引能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU2牵引能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU3牵引能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU4牵引能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>ACM2能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>ACM3能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>ACM1能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>ACM4能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Num7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>570</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num0">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>570</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>530</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>570</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>570</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>530</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>570</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_15">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU1电阻制动能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_16">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU2电阻制动能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_17">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU3电阻制动能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_18">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU4电阻制动能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_19">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU4再生制动能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_20">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU2再生制动能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_21">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU1再生制动能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_22">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>DCU3再生制动能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_24">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>480</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>TC2空压机运行时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_25">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:black;
font:20px  &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>TC1空压机运行时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="InputRecordtimeBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;color:rgb(30,200,200);border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputDistanceBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputTEDCU1Btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputTEDCU2Btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputTEDCU3Btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputTEDCU4Btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputSESIV2Btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputSESIV3Btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputSESIV4Btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputSESIV1Btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputREDCU3Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>320</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputBEDCU2Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputACP1RuntimeBtn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>420</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputBEDCU4Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputREDCU1Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputREDCU2Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>270</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputREDCU4Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>370</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputBEDCU3Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputBEDCU1Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="InputACP2RuntimeBtn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>470</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>530</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>设定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NumC">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>550</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>lbl1</zorder>
  <zorder>lbl1_2</zorder>
  <zorder>lbl1_3</zorder>
  <zorder>lbl1_4</zorder>
  <zorder>lbl1_5</zorder>
  <zorder>lbl1_6</zorder>
  <zorder>lbl1_7</zorder>
  <zorder>lbl1_8</zorder>
  <zorder>lbl1_9</zorder>
  <zorder>lbl1_10</zorder>
  <zorder>Num7</zorder>
  <zorder>Num4</zorder>
  <zorder>Num0</zorder>
  <zorder>Num1</zorder>
  <zorder>Num5</zorder>
  <zorder>Num6</zorder>
  <zorder>Num2</zorder>
  <zorder>Num8</zorder>
  <zorder>Num3</zorder>
  <zorder>Num9</zorder>
  <zorder>lbl1_15</zorder>
  <zorder>lbl1_16</zorder>
  <zorder>lbl1_17</zorder>
  <zorder>lbl1_18</zorder>
  <zorder>lbl1_19</zorder>
  <zorder>lbl1_20</zorder>
  <zorder>lbl1_21</zorder>
  <zorder>lbl1_22</zorder>
  <zorder>lbl1_24</zorder>
  <zorder>lbl1_25</zorder>
  <zorder>InputRecordtimeBtn</zorder>
  <zorder>InputDistanceBtn</zorder>
  <zorder>InputTEDCU1Btn</zorder>
  <zorder>InputTEDCU2Btn</zorder>
  <zorder>InputTEDCU3Btn</zorder>
  <zorder>InputTEDCU4Btn</zorder>
  <zorder>InputSESIV2Btn</zorder>
  <zorder>InputSESIV3Btn</zorder>
  <zorder>InputSESIV4Btn</zorder>
  <zorder>InputSESIV1Btn</zorder>
  <zorder>InputREDCU3Btn</zorder>
  <zorder>InputBEDCU2Btn</zorder>
  <zorder>InputACP1RuntimeBtn</zorder>
  <zorder>InputBEDCU4Btn</zorder>
  <zorder>InputREDCU1Btn</zorder>
  <zorder>InputREDCU2Btn</zorder>
  <zorder>InputREDCU4Btn</zorder>
  <zorder>InputBEDCU3Btn</zorder>
  <zorder>InputBEDCU1Btn</zorder>
  <zorder>InputACP2RuntimeBtn</zorder>
  <zorder>ConfirmBtn</zorder>
  <zorder>NumC</zorder>
  <zorder>returnBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
